server/models/Request.js

```diff
@@ -1,35 +1,32 @@
 const mongoose = require("mongoose");
 
 const requestSchema = new mongoose.Schema({
-    user_id:{
+    userId:{
         type: String,
         required: true,
       },
-    sitter_id: {
+    sitterId:{
         type: String,
         required: true,
     },
-    start: {
+    start:{
         type: Date,
         required: true
     },
     end:{
         type: Date,
-        required: true
-    },
-    accepted:{
-        type: Boolean,
-        default: False
+        required: true,
     },
-    declined: {
-        type: Boolean,
-        default: False
+    status:{
+        type: String,
+        enum : ["accepted", "declined"],
+        default: "declined"
     },
-    paid: {
+    paid:{
         type: Boolean,
-        default: False
+        default: false
     }
 });
 
-module.exports = Request = mongoose.model("request",requestSchema);
+module.exports = RequestModel = mongoose.model("request",requestSchema);
```

Comment:
Sorry about the confusion here - the `Request` name for exports is fine. We should modify the name we provide in `mongoose.model()` call.

Reply:
gotcha